src/components/DocumentSlicedName.vue

```diff
@@ -98,7 +106,7 @@ export default {
       return { id: this.document.source.rootDocument || this.document.id }
     },
     baseComponent() {
-      return this.hasActiveTextTruncate ? ActiveTextTruncate : 'span'
+      return this.hasActiveTextTruncate() ? ActiveTextTruncate : 'span'
```

Comment:
`hasActiveTextTruncate` looks like a computed property to me

Reply:
It's not, hence the confusion before! We should probably refactor this. This change here is just a bug fix.